_data/identity.yml

```diff
@@ -1,4 +1,15 @@
 websites:
+    - name: 1Password
+      url: https://1password.com/
+      img: 1password.png
+      tfa: No
+      email_address: support@1password.com
+      facebook: 1Password
+      twitter: 1Password
+      exceptions:
+        text: "AgileBits, the company behind 1Password, feels that two factor authentication doesn't really fit into 1Password's offering. We disagree because of the way in which 1Password handles authentication and password data."
```

Comment:
True.  My main issue with not acknowledging the argument that they made is that there was obviously a difference of opinion as to whether or not tfa was important for them.  I disagreed with their viewpoint; but I think that's something that people should have been able to decide for themselves, and acknowledging that there was a different viewpoint would have done the trick.  I guess since they are implementing it now, they've pretty much conceded their previous stance.